Feature/improve readme files (#73)
secureCodeBox/secureCodeBox-v2-alpha: hooks/persistence-defectdojo/README.md

---
title: "DefectDojo"
path: "hooks/persistence-defectdojo"
category: "hook"
type: "persistenceProvider"
state: "roadmap"

> usecase: "Publishes all Scan Findings to elasticsearch (ECK)."

Reviewer: copy paste error

```suggestion
usecase: "Publishes all Scan Findings to DefectDojo."
```

---

<!-- end -->

## About

> DefectDojo is an OpenSource Tools for importing and managing findings of security scanner. The DefectDojo persistence provider can be used to create new Engagements for SecurityTests run via the secureCodeBox and import all findings which were identified.

Reviewer: ```suggestion
DefectDojo is an open-source tool for importing and managing findings of security scanners. The DefectDojo persistence provider can be used to create new Engagements for SecurityTests run via the secureCodeBox and import all findings which were identified automatically to DefectDojo.
```

> Tools which are supported by the secureCodeBox and DefectDojo (OWASP ZAP, Arachni & Nmap) this is done by importing the raw scan report into DefectDojo. Findings by other secureCodeBox supported scanners are currently not directly supported by DefectDojo. These findings are imported via a generic finding api of defectDojo, which might cause some loss of information about the findings.  

Reviewer: ```suggestion
Tools which are supported both by the secureCodeBox and DefectDojo (OWASP ZAP & Nmap) this is done by importing the raw scan report into DefectDojo. Findings by other secureCodeBox supported scanners are currently not directly supported by DefectDojo. These findings are imported via a generic finding API of DefectDojo, which might cause some loss of information on the findings.  
```
